### PR TITLE
regenerate doc for cue schema 0.1.2720+053846c

### PR DESCRIPTION
**Pull request by tinajyhou**
regenerate data sharing public doc for updated CUE version 0.1.2720+053846c

### CODE REVIEW

File: assets/download_file/data-sharing-raw-table-schemas.txt [hunk 1]
```diff
@@ -1020,77 +1020,6 @@ USERS_MESSAGES_INAPPMESSAGE_IMPRESSION_SHARED: when a user views an in app messa
 	AD_TRACKING_ENABLED (BOOLEAN)
 	SF_CREATED_AT (TIMESTAMP_LTZ(9))
 
-USERS_MESSAGES_NEWSFEEDCARD_ABORT_SHARED: An originally scheduled newsfeedcard message was aborted for some reason.
```
Comment (jayhahn) [inline assets/download_file/data-sharing-raw-table-schemas.txt:1023]: This is incorrect, we haven't removed this. I think the command that generates this needs to be updated.

Comment (tinajyhou) [reply assets/download_file/data-sharing-raw-table-schemas.txt:1023]: https://github.com/Appboy/currents-users-events/commit/163970a15a764a69bdb33d8945929ca08341d329#diff-1e2551babd5a515628c64aab581b22b8ea00ad94d8a46fa960b32ea700bffe4cL725
isn't this removed?

Comment (jayhahn) [reply assets/download_file/data-sharing-raw-table-schemas.txt:1023]: They kept it in Datalake

**Comment by josh-mccrowell-braze** [comment]
Hi @tinajyhou : Please tag me as a reviewer when this one is ready to go live. Thanks!

File: assets/download_file/data-sharing-raw-table-schemas.txt (hunk 1 above)
Comment (jayhahn) [reply assets/download_file/data-sharing-raw-table-schemas.txt:1023]: This is fixed https://github.com/Appboy/currents-users-events/pull/667 so I think we can just close this PR?

Comment (tinajyhou) [reply assets/download_file/data-sharing-raw-table-schemas.txt:1023]: yup closing this PR, there's no change in this case

**Pull request closed without merging**